midend/interpreter.h

```diff
@@ -420,7 +420,7 @@ class SymbolicHeaderUnion : public SymbolicStruct {
     SymbolicBool* valid = nullptr;
     SymbolicHeaderUnion(const IR::Type_HeaderUnion* type, bool uninitialized,
                    const SymbolicValueFactory* factory);
-    virtual void setValid(bool v);
+    virtual void setValid(bool v, cstring field = nullptr);
```

Comment:
What does it mean for the field to be nullptr? I would not allow this.

Reply:
This is necessary to check and separate the setValid call for the entire HeaderUnion or for its field

Reply:
You can not really set a header union valid or invalid, since that depends on its members. So similarly, we should not need this.